pkg/session/bootstrap.go

```diff
@@ -730,6 +730,29 @@ const (
 		update_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
 		PRIMARY KEY (restore_id, restored_ts, upstream_cluster_id, segment_id));`
 
+	// CreateRestoreRegistryTable is a table that tracks active restore tasks to prevent conflicts.
+	CreateRestoreRegistryTable = `CREATE TABLE IF NOT EXISTS mysql.tidb_restore_registry (
+		id BIGINT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+		filter_strings TEXT NOT NULL,
+		filter_hash VARCHAR(64) NOT NULL,
+		start_ts BIGINT UNSIGNED NOT NULL,
+		restored_ts BIGINT UNSIGNED NOT NULL,
+		upstream_cluster_id BIGINT UNSIGNED,
+		with_sys_table BOOLEAN NOT NULL DEFAULT TRUE,
+		status VARCHAR(20) NOT NULL DEFAULT 'running',
+		cmd TEXT,
+		start_timestamp BIGINT UNSIGNED NOT NULL,
```

Comment:
From a user perspective I find the column naming confusing.

First, it is not clear how this is different from `start_ts` referenced above. Can you clarifying the naming here? If they are used for different things, is the a better name that can be used to make it more obvious? Or is this just intended to offer different levels of precision (if so why the same data type)?

Second, the suffix of the column name `_timestamp` implies this is a`TIMESTAMP` while the type is specified as `BIGINT`. This is confusing and inconsistent.